src/components/manage/Widgets/SelectWidget.jsx

```diff
@@ -213,8 +216,11 @@ class SelectWidget extends Component {
     const { id, choices, onChange } = this.props;
     // Make sure that both disabled and isDisabled (from the DX layout feat work)
     const disabled = this.props.disabled || this.props.isDisabled;
+    const isMulti = this.props.isMulti || id === 'roles' || id === 'groups';
     const Select = this.props.reactSelect.default;
-    const AsyncPaginate = this.props.reactSelectAsyncPaginate.AsyncPaginate;
+    const AsyncPaginate =
+      this.props.reactSelectAsyncPaginate.AsyncPaginate ||
+      this.props.reactSelectAsyncPaginate.default;
```

Comment:
Javascript bundling... I've learned that our modules are transpiled by babel to cjs, so that `default` is just an artefact of that. I don't know why it's needed in some places... hopefully at some point this whole toolchain gets cleaned up to a more consistent state.